src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,98 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.Comparator;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] data = getSortedData(getSumItems(fromFileName));
+        StringBuilder resultData = new StringBuilder();
+        for (String line : data) {
+            resultData.append(line).append(System.lineSeparator());
+        }
+        resultData.append(getOdd(data));
+        writeToFile(resultData.toString().split(System.lineSeparator()), toFileName);
+    }
+
+    private String[] getSortedData(String[] resultData) {
```

Comment:
The 'getSortedData' method should be using a lambda expression for the Comparator to make the code more concise and readable.